net/nfs-kernel-server/Makefile

```diff
@@ -208,19 +221,53 @@ define Package/nfs-utils/InstallDev
 	$(CP) $(PKG_INSTALL_DIR)/usr/lib/pkgconfig/libnfsidmap.pc $(1)/usr/lib/pkgconfig/
 endef
 
+define Package/nfs-kernel-server/install
+	$(INSTALL_DIR) $(1)/etc/init.d $(1)/etc/config $(1)/usr/sbin
```

Comment:
There aren't any files placed in `/etc/config/`, so that path should be removed.